File: CSharp/DP.Domain/Samples/Bridge/PrinterCustom.cs
using DP.Domain.Samples.Bridge;

namespace DP.Domain.Samples.Bridge
{
    public class PrinterCostom : IPrinter
    {
        private IPrintStg _printStg = null;
        public PrinterCostom(IPrintStg printStg)
        {
            this._printStg = printStg;
        }

        public void OrderA()
        {
            this._printStg.PrintA();
        }

        public void OrderB()
        {
            this._printStg.PrintB();
        }
    }
}

File: Python/Samples/Bridge/PrinterCustom.py
from Printers import Printer
from PrintStgs import *

class PrinterCustom(Printer):
    _printStg=None

    def __init__(self, printStg=PrintStg):
        if printStg is None:
            raise TypeError
        else:     
            self._printStg = printStg
    
    def orderA(self):
        self._printStg.printA()

    def orderB(self):
        self._printStg.printB()